src/main/java/com/squareup/kotlinpoet/PropertySpec.kt

```diff
@@ -43,7 +43,9 @@ class PropertySpec private constructor(builder: Builder) {
     val isInlineProperty = getter?.modifiers?.contains(KModifier.INLINE) ?: false &&
         setter?.modifiers?.contains(KModifier.INLINE) ?: false
     val propertyModifiers = if (isInlineProperty) modifiers + KModifier.INLINE else modifiers
-    codeWriter.emitKdoc(kdoc)
+    if (!inline) {
```

Comment:
Curious: why the inline check?

Reply:
`inline` will only be true when properties are used as primary constructor params, in that case there should be no new line after the property code and property annotations should be printed on a single line. The name is confusing though.